src/test/mock/SimpleStateBridge.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {IBridge} from "../../interfaces/IBridge.sol";
+
+/// @title SimpleStateBridge
```

Comment:
I think the title is meant to be human readable.

Reply:
what do you mean by human readable?